packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -57,23 +68,49 @@ const hasNoAsyncOpOrExists = async (cumulusMessage, knex) => {
   }, knex, tableNames.asyncOperations);
 };
 
-const hasNoCollectionOrExists = async (cumulusMessage, knex) => {
-  const collectionInfo = getCollectionNameAndVersionFromMessage(cumulusMessage);
-  if (!collectionInfo) {
-    return true;
+const getMessageCollection = async (cumulusMessage, knex) => {
```

Comment:
Executions, PDRs, and granules all need to refer to the collection in RDS (if it exists), so it made sense to me to convert `hasNoCollectionOrExists` into `getMessageCollection`, which actually returns the collection rather than just saying whether it exists.